duet/duet.model.lkml

```diff
@@ -4,6 +4,18 @@ include: "views/*"
 include: "explores/*"
 include: "dashboards/*"
 
+##########################################################################################
+# below are views saved into the shong-dev directory specifically for the MR1 dashboard
+# do not use those views for other projects, as they are specific to the MR1 dashboard and
+# haven't been developed for general consumption
+include: "/shong-dev/views/version_uplift.view.lkml"
+include: "/shong-dev/views/surveys.view.lkml"
+include: "/shong-dev/views/feature_usage.view.lkml"
+include: "/shong-dev/views/gplay_downloads.view.lkml"
+include: "/shong-dev/views/mozorg_ga.view.lkml"
+include: "/shong-dev/views/mozblogs_ga.view.lkml"
```

Comment:
You can do: `include: "/shong-dev/views/*.view.lkml"`

Reply:
fixed directory structure